container_main.py

```
@@ -66,19 +66,28 @@ def run_scheduler():
         os.environ["FLASK_DEBUG"] = "false"
-        os.environ["PORT"] = os.environ.get("PORT", "5000")
-
-        # Import and run the API server
-        from api_server import app
-
-        port = int(os.environ.get("PORT", 5000))
-        app.run(host="0.0.0.0", port=port, debug=False, use_reloader=False)
+        port = os.environ.get("PORT", "5000")
+
+        # Use gunicorn to run the Flask app in production
+        subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)